static/js/collapsible-filters/interactions.js

```diff
@@ -240,4 +241,56 @@ export default class Interactions {
       yxtFooter.classList.add(this.templateName);
     }
   }
+
+  /**
+   * 
+   * Use media query with $breakpoint-expanded-filters to determines if the
+   * page is in collapsible filters view.
+   * 
+   * @returns {boolean} whether the viewport is in collapsible filters view
+   */
+  isCollapsibleFiltersView() {
+    const mediaQuery = window.matchMedia(`(min-width: ${variables.cssBreakpointExpandedFilters})`);
+    return !mediaQuery.matches;
+  }
+
+  /**
+   * Add a media query event listener when there's a width view port change
+   * relative to $breakpoint-expanded-filters.
+   *
+   * @param {Function} handleViewportChange callback function to invoke when there's a media query event
+   */
+  addExpandedFiltersListener(handleViewportChange) {
+    const mediaQuery = window.matchMedia(`(min-width: ${variables.cssBreakpointExpandedFilters})`);
+    if ('addEventListener' in mediaQuery) {
+      mediaQuery.addEventListener('change', (e) => handleViewportChange(e.matches))
+     } else if ('addListener' in mediaQuery) {
+      mediaQuery.addListener(e => handleViewportChange(e.matches))
+     }
+  }
+
+  /**
+   * Update FiltersWrapper HTML element's location when switching between
+   * expanded and collapsible filters view in page templates that follows
+   * the left-mid-right div structure (e.g. VerticalStandard and VerticalGrid).
+   */
+  setupFiltersWrapper() {
```

Comment:
Could we make this name a bit more descriptive? Maybe use the word "responsive" somewhere